api/mysagw/identity/migrations/0008_public_localized_interests.py

```diff
@@ -0,0 +1,112 @@
+# Generated by Django 2.2.20 on 2021-09-20 08:07
+
+from django.db import migrations, models
+import localized_fields.fields.char_field
+
+
+def set_all_title_hstore(apps, schema_editor):
+    models = [
+        apps.get_model("identity", "Interest"),
+        apps.get_model("identity", "HistoricalInterest"),
+        apps.get_model("identity", "InterestCategory"),
+        apps.get_model("identity", "HistoricalInterestCategory"),
+    ]
+    for model in models:
+        for record in model.objects.iterator():
+            record.title_hstore = {"de": record.title}
+            record.save()
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("identity", "0007_organisation_types"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="historicalinterestcategory",
+            name="public",
+            field=models.BooleanField(default=False),
+        ),
+        migrations.AddField(
+            model_name="interestcategory",
+            name="public",
+            field=models.BooleanField(default=False),
+        ),
+        migrations.AddField(
+            model_name="interestcategory",
+            name="title_hstore",
+            field=localized_fields.fields.char_field.LocalizedCharField(
+                blank=True, null=True, required=[]
+            ),
+        ),
+        migrations.AddField(
+            model_name="interest",
+            name="title_hstore",
+            field=localized_fields.fields.char_field.LocalizedCharField(
+                blank=True, null=True, required=[]
+            ),
+        ),
+        migrations.AddField(
+            model_name="historicalinterestcategory",
+            name="title_hstore",
+            field=localized_fields.fields.char_field.LocalizedCharField(
+                blank=True, null=True, required=[]
+            ),
+        ),
+        migrations.AddField(
+            model_name="historicalinterest",
+            name="title_hstore",
+            field=localized_fields.fields.char_field.LocalizedCharField(
+                blank=True, null=True, required=[]
+            ),
+        ),
+        migrations.RunPython(set_all_title_hstore, migrations.RunPython.noop),
```

Comment:
I'm surprised this works :-D In my experience in the past, Django tries to "simplify" migration steps if done within the same migration, breaking things if mixing schema changes with `RunPython` bits. Consider me happy to see this works